versions-of-the-bot/premium.md

```diff
@@ -29,18 +29,20 @@ This tier is personal and doesn't grant access to the MoonlightBot premium bot.
 * MoonlightBot premium for one server
 * Allow the addition of up to three collaborators
 * Feature requests with higher priority
+* The ability to buy up to two gift subscriptions at half price
```

Comment:
I'm a bit skeptical about having those inside the tier perk description... I feel this contradicts the "everything from the previous tiers" later on because you cannot gift if you're on a lifetime subscription

Reply:
That makes sense. It's gone